.github/workflows/release.yml

```diff
@@ -1,38 +1,35 @@
-name: Release
+name: Release to PyPI
 
 on:
   release:
     types: [released]
 
-# set environment variables
 env:
   TOKEN: ${{ secrets.GH_TOKEN }}
   POETRY_HTTP_BASIC_PYPI_USERNAME: ${{ secrets.PYPI_USER }}
   POETRY_HTTP_BASIC_PYPI_PASSWORD: ${{ secrets.PYPI_PW }}
 
 jobs:
-  # release to PyPI
   release-pypi:
-    name: Release to PyPI
+    name: release-to-pypi
```

Comment:
Is there a reason why you changed the name here to kebab case?

Reply:
I tried to harmonize the naming convention across the different YML files. But motivated by your comment, I had a 2nd look at the names and found out that some of them aren't even necessary, and can be removed, because the name is already defined by the YAML-key.